napari/plugins/_skimage_data.py

```diff
@@ -0,0 +1,58 @@
+from napari_plugin_engine import napari_hook_implementation
+
+# fmt: off
+SKIMAGE_DATA = [
+    'astronaut', 'binary_blobs', 'brain', 'brick', 'camera', 'cat', 'cell',
+    'cells3d', 'checkerboard', 'clock', 'coffee', 'coins', 'colorwheel',
+    'eagle', 'grass', 'gravel', 'horse', 'hubble_deep_field', 'human_mitosis',
+    'immunohistochemistry', 'kidney', 'lfw_subset', 'lily', 'microaneurysms',
+    'moon', 'page', 'retina', 'rocket', 'shepp_logan_phantom', 'skin', 'text'
+]
+# fmt: on
+
+
+def _load_skimage_data(name):
+    import skimage.data
```

Comment:
@jni does the skimage data now use pooch? is it possible that might get pulled out one day into it's own thing?

I guess we were saying that once skimage drops matplotlib we might want to add it back as a dependency?

I like the idea of making this always available to users

Reply:
yes we use pooch. Indeed, if we merge this, it might be good to add `scikit-image[data]` to `napari[all]`. People who want a more minimal install can always do `napari[pyqt]`?

Reply:
> might be good to add scikit-image[data] to napari[all]

:heart: